tests/shims/mediadevices.js

```diff
@@ -0,0 +1,280 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const getMediaDevicesInstance = require('../../lib/twilio/shims/mediadevices');
+
+describe('MediaDevicesShim', () => {
+  const userMediaStream = 'USER-STREAM';
+
+  let clock;
+  let globalMediaDevices;
+  let getDevices;
+  let mediaDevices;
+  let mediaDeviceList;
+  let nativeMediaDevices;
+
+  const sampleDevices = async (count = 1) => {
```

Comment:
```suggestion
  const sampleDevices = async () => {
```
Might be missing something, but is this used anywhere?

Reply:
Nope. I was using it before, but it needs to be removed now.